app/models/item.rb

```diff
@@ -19,6 +19,22 @@ def self.disabled_only
     where(enabled: false)
   end
 
+  def self.top_5_by_revenue 
+    joins(:invoice_items)
+    .joins('right join invoices on invoices.id=invoice_items.invoice_id')
+    .joins('right join transactions on transactions.invoice_id=invoices.id')
+    .select('items.*, sum(invoice_items.quantity * invoice_items.unit_price) as revenue')
+    .where(transactions: {result: 1})
```

Comment:
@mkrumholz Could you do `.where(transactions.result => 1, invoices.status =>1)`?

Reply:
Will update -- you don't need the hash rockets, though. Just plain `transactions: {result: 1}, invoices: {status: 1}`

Reply:
I just don't like hash rockets. Rockets belong in space not code 🚀